main.go

```diff
@@ -18,6 +18,13 @@ func main() {
 		os.Exit(1)
 	}
 
+	migrationsDir := "./migrations" 
```

Comment:
We were going to remove the ./migrations and use cmd/migrations, as we were going to let bun handle it?